php/WP_CLI/CommandWithUpgrade.php

```diff
@@ -10,6 +10,13 @@
 	protected $upgrade_refresh;
 	protected $upgrade_transient;
 
+	function __construct() {
+		// Do not automatically check translations updates after updating plugins/themes.
```

Comment:
Same as #1501? Can you comment on the core issue?

Seems pretty dirty to add a filter inside of `__construct()` — should this always run, or just on specific contexts?